Apps/WebClient/src/ClientApp/app/models/laboratoryTimelineEntry.ts

```diff
@@ -80,7 +82,7 @@ export class LaboratoryResultViewModel {
   constructor(model: LaboratoryResult) {
     this.id = model.id;
     this.testType = model.testType;
-    this.outOfRange = model.outOfRange;
+    this.outOfRange = model.outOfRange ? "True" : "False";
```

Comment:
This should be a a view level change and not a model change.

Reply:
thisis the view model

Reply:
I'll give it to your for the smiley!  Most of the other code has it mapped in the actual view and not in the model.